pkg/api/login_test.go

```diff
@@ -131,10 +140,10 @@ func TestLoginErrorCookieAPIEndpoint(t *testing.T) {
 	}
 	sc.m.Get(sc.url, sc.defaultHandler)
 	sc.fakeReqNoAssertionsWithCookie("GET", sc.url, cookie).exec()
-	assert.Equal(t, sc.resp.Code, 200)
+	require.Equal(t, 200, sc.resp.Code)
```

Comment:
does this mean we always want to use require?
`Package require implements the same assertions as to the `assert` package but stops test execution when a test fails.`
Or we just want to use require when checking the null pointer then stop testing when the pointer is null so no point to continue?

Reply:
@ying-jeanne It doesn't mean we always want to use `require`, but my thinking here is that the test should stop if an error response is received, because the remaining checks don't make any sense.

Reply:
@ying-jeanne If `require.Equal` fails here, the execution of that particular test will end, yes. The rest of the test won't make any sense anyhow.